modules/fedimint-wallet/src/lib.rs

```diff
@@ -1090,7 +1093,7 @@ impl<'a> StatelessWallet<'a> {
 
             let signature = self
                 .secp
-                .sign_ecdsa(&Message::from_slice(&tx_hash[..]).unwrap(), &tweaked_secret);
+                .sign_ecdsa(&Message::from_slice(&tx_hash[..])?, &tweaked_secret);
```

Comment:
Should be an `expect` or maybe even use the infallible [`ThirtyTwoByteHash`](https://docs.rs/secp256k1/0.24.0/secp256k1/trait.ThirtyTwoByteHash.html) API.